skycfg.go

```diff
@@ -86,9 +86,8 @@ func (r *localFileReader) ReadFile(ctx context.Context, path string) ([]byte, er
 
 // NewProtoMessage returns a Starlark value representing the given Protobuf
 // message. It can be returned back to a proto.Message() via AsProtoMessage().
-func NewProtoMessage(msg proto.Message) starlark.Value {
-	v, _ := protomodule.NewMessage(msg)
-	return v
+func NewProtoMessage(msg proto.Message) (starlark.Value, error) {
```

Comment:
Since this is a backwards-incompatible change, are you going to release these changes as a new (major?) version for skycfg?

Reply:
Yes, as I understand the current state is Skycfg has a single release tagged 0.1.0 and there have been a number of other API changes. Also as we learned in https://github.com/stripe/skycfg/pull/95 API changes are easier to deal with than implementation changing untested behavior